packages/ag-charts-community/src/scene/group.ts

```diff
@@ -81,42 +100,102 @@ export class Group extends Node {
     }
 
     override render(renderCtx: RenderContext) {
-        const { layer } = this;
+        const { layer, renderToOffscreenCanvas } = this;
         const childRenderCtx: RenderContext = { ...renderCtx };
 
-        if (layer == null) {
+        if (!renderToOffscreenCanvas) {
             this.renderInContext(childRenderCtx);
             super.render(childRenderCtx); // Calls markClean().
             return;
         }
 
-        const { ctx, stats } = renderCtx;
+        const { ctx, stats, devicePixelRatio: pixelRatio } = renderCtx;
 
+        let { image } = this;
         if (this.isDirty(renderCtx)) {
-            const transform = ctx.getTransform();
-            const { globalAlpha } = ctx;
+            image?.bitmap.close();
+            image = undefined;
+
+            const bbox = layer ? BBox.NaN : this.computeBBox();
+
+            const renderOffscreen = (
+                offscreenCtx: OffscreenCanvasRenderingContext2D,
+                ...transform: [DOMMatrix] | [number, number, number, number, number, number]
+            ) => {
+                childRenderCtx.ctx = offscreenCtx;
+                offscreenCtx.save();
+                offscreenCtx.setTransform(...(transform as any[]));
+                offscreenCtx.globalAlpha = 1;
+                this.renderInContext(childRenderCtx);
+                offscreenCtx.restore();
+                (offscreenCtx as any).verifyDepthZero?.(); // Check for save/restore depth of zero!
+            };
```

Comment:
I have some concerns about affecting this performance-sensitive code path, and specifically about introducing lambda functions here with that in mind - do we need to merge the sprint rendering functionality in here?